framework/src/main/java/org/checkerframework/framework/util/FieldInvariants.java

```diff
@@ -83,6 +83,9 @@ public FieldInvariants(
             throw new BugInCF("malformed FieldInvariants");
         }
         String fieldString = field.toString();
+        if (!isWellFormed()) {
```

Comment:
This repeats the check from a few lines above and nothing of the state seems to have changed in between. Is this redundant?